### PR TITLE
Rename Amazon Ads | Fixed the refresh access token issue and some minor improvments.

### PR DESCRIPTION
**Pull request by Innovative-GauravKochar**
This Pull request is to rename Amazon Ads ,fixing the refresh access token issue and some minor improvements.
Jira ticket:- https://segment.atlassian.net/jira/software/c/projects/STRATCONN/boards/310?assignee=712020%3A283b30db-5958-468c-86a7-1c580ddd1850&assignee=63617339fc0cc7a600b03c6b&selectedIssue=STRATCONN-3841

## Testing

_Include any additional information about the testing you have completed to
ensure your changes behave as expected. For a speedy review, please check
any of the tasks you completed below during your testing._

- [ ] Added [unit tests](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing) for new functionality - Not required
- [x] Tested end-to-end using the [local server](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing)
- [ ] [Segmenters] Tested in the staging environment - Not required

Screenshot 2024-06-05 at 7 24 54 PM

Screenshot 2024-06-06 at 2 23 43 PM

### CODE REVIEW

File: packages/destination-actions/src/destinations/amazon-amc/index.ts [hunk 1]
```diff
@@ -126,7 +124,23 @@ const destination: AudienceDestinationDefinition<Settings, AudienceSettings> = {
     currency: {
       label: 'Currency',
       type: 'string',
-      description: `The price paid. Base units depend on the currency. As an example, USD should be reported as Dollars.Cents, whereas JPY should be reported as a whole number of Yen. All provided values will be rounded to two digits with toFixed(2).Refer [Aamzon Ads Documentation](https://advertising.amazon.com/API/docs/en-us/amc-advertiser-audience#tag/Audience-Metadata/operation/CreateAudienceMetadataV2) to view supported Currency`
+      description: `Currency code for the CPM value.`,
+      choices: [
```
Comment (varadarajan-tw) [inline packages/destination-actions/src/destinations/amazon-amc/index.ts:128]: Does this dropdown work in UI? I thought it didn't?

File: packages/destination-actions/src/destinations/amazon-amc/index.ts [hunk 2]
```diff
@@ -92,8 +91,7 @@ const destination: AudienceDestinationDefinition<Settings, AudienceSettings> = {
     return {
       headers: {
         authorization: `Bearer ${auth?.accessToken}`,
-        'Amazon-Advertising-API-ClientID': process.env.ACTIONS_AMAZON_ADS_CLIENT_ID || '',
-        'Content-Type': 'application/vnd.amcaudiences.v1+json'
```
Comment (varadarajan-tw) [inline packages/destination-actions/src/destinations/amazon-amc/index.ts:96]: Would removing Content-Type affect the API in sync call?

File: packages/destination-actions/src/destinations/amazon-amc/index.ts (hunk 1 above)
Comment (Innovative-GauravKochar) [reply packages/destination-actions/src/destinations/amazon-amc/index.ts:128]: Yes ,it worked and tested in staging

File: packages/destination-actions/src/destinations/amazon-amc/index.ts (hunk 2 above)
Comment (Innovative-GauravKochar) [reply packages/destination-actions/src/destinations/amazon-amc/index.ts:96]: No , we specified content-type in upload record api  call in Sync audiences seperately, Should not impact any other api.

**Pull request merged**